login-workflow/docs/AcceptEula.md

```diff
@@ -0,0 +1,67 @@
+# Accept Eula
+
+The `<AcceptEula>` component that renders a screen displaying the EULA and requests acceptance via a checkbox.
+
+<img width='100%' alt="Accept Eula" src="../media/accept_eula.png">
+
+## Usage
+
+```tsx
+import { useState } from 'react';
+import { AcceptEula } from '@brightlayer-ui/react-auth-workflow';
+
+const [eulaAccepted, setEulaAccepted] = useState(false);
+
+<AcceptEula 
+    eulaAccepted={eulaAccepted} 
+    onEulaChanged={setEulaAccepted} 
+    loadEula={()=>{ // ...Load the Eula if we do not yet have the content }} 
+/>
+```
+
+## API
+
+<div style="overflow: auto">
+
+| Prop Name                | Description                                                | Type                                                                                                     | Required | Default                                           |
+| ------------------------ | ---------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- | -------- | ------------------------------------------------- |
+| eulaAccepted             | Prop to check the checkbox                                 | `boolean`                                                                                                | yes      |                                                   |
+| eulaContent              | The content to render for the EULA                         | `string`                                                                                                 | no       |                                                   | 
+| onEulaChanged            | Function to call when the state of the checkbox is changed | `function`                                                                                               | yes      |                                                   |
+| loadEula                 | Function to call to retrieve the eulaContent               | `function`                                                                                               | yes      |                                                   |
+| htmlEula                 | Prop to render EULA as HTML                                | `boolean`                                                                                                | no       |                                                   |
+| eulaError                | Error message if the EULA fails to load                    | `string`                                                                                                 | no       |                                                   |
```

Comment:
This prop name is unclear.  Is `eulaError` a text? A callback function triggered whenever there is an error? To pass in a JavaScript `Error` object? Users are expected to mostly  guess what a prop does based on just the way it is named.

Reply:
There are a lot of similar issues in the current API table. I did not highlight all of them; I think you can infer what kind of changes I am looking for based on my other suggestions. I recommend you applying design thinking principles you learned a while ago and doing a user testing activity with Manoj. You provide only the prop name, and let him guess what the prop is controlling.

Reply:
> This prop name is unclear. Is `eulaError` a text? A callback function triggered whenever there is an error? To pass in a JavaScript `Error` object? Users are expected to mostly guess what a prop does based on just the way it is named.

Yes eulaError is string set to transitErrorMessage